### PR TITLE
Security Article for Makers

### PR DESCRIPTION
**Pull request by lthiery**
addresses #171

### CODE REVIEW

**Comment by lthiery** [comment]
I snuck in a bit of formatting and added them to `.git-blame-ignore-revs`, so please do not squash and merge.

File: docs/mine-hnt/hotspot-makers/security-requirements/security-requirements.mdx [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+id: security-requirements
+hide_title: true
+sidebar_label: Docker Integration
+slug: /mine-hnt/hotspot-makers/security-requirements
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Security Requirements
```
Comment (PharkMillups) [inline docs/mine-hnt/hotspot-makers/security-requirements/security-requirements.mdx]:
```suggestion
## Security Requirements
```

File: docs/mine-hnt/hotspot-makers/security-requirements/security-requirements.mdx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+at this time. This helps insure that the Miner is running on real hardware and
+increases the barriers to dishonest actors.
+
+While formalizing the audit process, DeWi has determined that the only approved
+approach to guaranteeing this at this time is using the ECC608 to generate and
+store the key during manufacturing. Other approaches are possible, but they
+would require case-by-case consideration and the potentially a third-party
+audit.
+
+# Securing the swarm_key with the ECC608
```
Comment (PharkMillups) [inline docs/mine-hnt/hotspot-makers/security-requirements/security-requirements.mdx]:
```suggestion
## Securing the swarm_key with the ECC608
```

File: docs/mine-hnt/hotspot-makers/security-requirements/security-requirements.mdx [hunk 3]
```diff
@@ -0,0 +1,116 @@
+---
+id: security-requirements
+hide_title: true
+sidebar_label: Docker Integration
```
Comment (PharkMillups) [inline docs/mine-hnt/hotspot-makers/security-requirements/security-requirements.mdx]:
```suggestion
sidebar_label: Security Requirements
```

File: docs/mine-hnt/hotspot-makers/security-requirements/security-requirements.mdx [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+slug: /mine-hnt/hotspot-makers/security-requirements
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Security Requirements
+
+When the Helium Community passed
+[HIP19](https://github.com/helium/HIP/blob/master/0019-third-party-manufacturers.md),
+is was made clear that open and easy access to the `swarm_key` is not permitted
```
Comment (PharkMillups) [inline docs/mine-hnt/hotspot-makers/security-requirements/security-requirements.mdx]:
```suggestion
it was made clear that open and easy access to the `swarm_key` is not permitted
```

Comment (PharkMillups) [reply docs/mine-hnt/hotspot-makers/security-requirements/security-requirements.mdx]: Can we add some sort of "info" box about what the swarm key actually is?

File: docs/mine-hnt/hotspot-makers/security-requirements/security-requirements.mdx [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+|     serial_num     |  ok  |
+|{zone_locked,config}|  ok  |
+| {zone_locked,data} |  ok  |
+|    slot_config     |  ok  |
+|     key_config     |  ok  |
+|     miner_key      |  ok  |
++--------------------+------+
+```
+
+And finally, you will want to extra the `onboarding_key` which is also called
```
Comment (PharkMillups) [inline docs/mine-hnt/hotspot-makers/security-requirements/security-requirements.mdx]:
```suggestion
And finally, you will want to extract the `onboarding_key` which is also called
```

**Review by PharkMillups — CHANGES_REQUESTED** [review]

**Review by PharkMillups — APPROVED** [review]

**Pull request merged**